pkg/validations/cluster_test.go

```diff
@@ -232,3 +234,106 @@ func TestValidateK8s126SupportActive(t *testing.T) {
 	os.Setenv(features.K8s126SupportEnvVar, "true")
 	tt.Expect(validations.ValidateK8s126Support(tt.clusterSpec)).To(Succeed())
 }
+
+func TestValidateManagementClusterBundlesVersion(t *testing.T) {
+	type testParam struct {
+		mgmtBundlesName   string
+		mgmtBundlesNumber int
+		wkBundlesName     string
+		wkBundlesNumber   int
+		wantErr           string
+		errGetEksaCluster error
+		errGetBundles     error
+	}
+
+	testParams := []testParam{
+		{
+			mgmtBundlesName:   "bundles-28",
+			mgmtBundlesNumber: 28,
+			wkBundlesName:     "bundles-27",
+			wkBundlesNumber:   27,
+			wantErr:           "",
+		},
+		{
+			mgmtBundlesName:   "bundles-28",
+			mgmtBundlesNumber: 28,
+			wkBundlesName:     "bundles-29",
+			wkBundlesNumber:   29,
+			wantErr:           "cannot upgrade workload cluster with bundle 29 while management cluster management-cluster is on older bundle 28",
```

Comment:
I'm not sure this error reads well with the double `management cluster management-cluster`.

Reply:
`management-cluster` is the name of the management cluster.